### ISSUE TITLE
Missing "dynamic import wrapper" section in README

### ISSUE THREAD

**Issue by micah-yeager** [extend-chrome/rollup-plugin-chrome-extension#68]
Documentation Is:

- [x] Missing
- [x] Needed
- [x] Confusing
- [ ] Not Sure?

### Please Explain in Detail...
In README.md, the following section mentions the dynamic importer for importing modules into the background.js page:
```
⭐️ Use ES2015 Modules In Your Scripts
Chrome extensions don't support modules in background and content scripts. We've developed a [module loader](https://github.com/extend-chrome/rollup-plugin-chrome-extension#dynamic-import-wrapper) specifically for Chrome extension scripts, so you can take advantage of Rollup's great code splitting features.
```

However, the link to the "dynamic import wrapper" anchor doesn't go anywhere on the page, and that section seems to be missing altogether in the README.

### Your Proposal for Changes

I'd love to see this portion of the README fleshed out, maybe with an example? I want to use this feature in my background.js, but I don't really know where to start — having this reference fixed would help those of us who are new to the project.

**Comment by jacksteamdev** [comment]
Hi @micah-yeager, thanks for the heads up on the broken link! 👍

I just updated the link and added more information on the [dynamic import wrapper](https://github.com/extend-chrome/rollup-plugin-chrome-extension#%EF%B8%8F-use-es2015-modules-in-your-scripts).

Let me know what your think! 

---
### ⭐️ Use ES2015 Modules In Your Scripts <a name = "features-modules"></a>

Chrome extensions don't support modules in background and content scripts. We've developed a [module loader](https://github.com/extend-chrome/rollup-plugin-chrome-extension/blob/master/API.md#options-dynamic-import-wrapper) specifically for Chrome extension scripts, so you can take advantage of Rollup's great code splitting features.

### ⭐️ How does the dynamic import wrapper work?
TLDR; RPCE parses the manifest and replaces any script path with an IIFE wrapper that loads the script using a [dynamic import statement](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#dynamic_imports).

> See [Why do we need to use dynamic import in scripts?](https://github.com/extend-chrome/rollup-plugin-chrome-extension/blob/master/API.md#why-do-we-need-to-use-dynamic-import-in-scripts) for the reasons we need dynamic imports.
Content scripts are pretty straight forward, and the wrapper they use is just an IIFE that loads the script and logs any import errors.
Background pages need a special wrapper because [wake events](https://developer.chrome.com/docs/extensions/mv2/background_pages/) (onInstalled, onMessage, etc...) fire before the import completes. We make sure we don’t miss any events by capturing all events that occur before the import promise resolves and then re-dispatch them after the import completes.

**Comment by micah-yeager** [comment]
That makes a lot more sense. So am I correct in thinking that it doesn't need anything special to work out of the box, just use normal `import` statements in the background script?

**Comment by jacksteamdev** [comment]
@micah-yeager That's right! (sorry for the long delay)
